infra/chunkedEncode/KChunkedEncode.php

```diff
@@ -1046,7 +1046,8 @@ public function getSessionName($mode="merged")
 		{
 			switch($mode){
 			case "merged":
-				$name = $this->setup->output."_merged";
+				$name = $this->params->output;
```

Comment:
was the usage of params intent?

Reply:
yes, it's the params saved on the class object